Wikipedia/Code/ArticleViewController+Editing.swift

```diff
@@ -247,24 +247,47 @@ extension ArticleViewController: EditorViewControllerDelegate {
         }
     }
     
-    func editorDidFinishEditing(_ editor: EditorViewController, result: Result<EditorChanges, Error>) {
+    func editorDidFinishEditing(_ editor: EditorViewController, result: Result<EditorChanges, Error>, needsNewTempAccountToast: Bool?) {
         switch result {
         case .failure(let error):
             showError(error)
         case .success(let changes):
             dismiss(animated: true) {
                 
                 let title = CommonStrings.editPublishedToastTitle
-                let image = UIImage(systemName: "checkmark.circle.fill")
+                let image = UIImage(named: "temp")
+                let tempAccountUsername = self.dataStore.authenticationManager.authStateTemporaryUsername ?? "*****"
                 
                 if UIAccessibility.isVoiceOverRunning {
                     DispatchQueue.main.asyncAfter(deadline: .now() + 2.0) {
                         UIAccessibility.post(notification: UIAccessibility.Notification.announcement, argument: title)
                     }
                 } else {
-                    WMFAlertManager.sharedInstance.showBottomAlertWithMessage(title, subtitle: nil, image: image, type: .custom, customTypeName: "edit-published", dismissPreviousAlerts: true)
+                    WMFAlertManager.sharedInstance.showBottomAlertWithMessage(
+                        title,
+                        subtitle: nil,
+                        image: image,
+                        type: .custom,
+                        customTypeName: "edit-published",
+                        dismissPreviousAlerts: true,
+                        completion: {
+                            if let needsNewTempAccountToast, needsNewTempAccountToast {
+                                let title = WMFLocalizedString("article-view-controller-editing-temp-account-created-title", value: "Temporary account created", comment: "After a user edits an article, creating an IP account, this pop-up title lets them know.")
+                                let format = WMFLocalizedString("article-view-controller-editing-temp-account-created-subtitle", value: "Temporary account %1$@ was created after your edit was published. It will expire in 90 days.", comment: "More information on the creation of temporary accounts, $1 replaces their username.")
+                                let subtitle = String.localizedStringWithFormat(format, tempAccountUsername)
+                                let image = UIImage(systemName: "checkmark.circle.fill")
+                                WMFAlertManager.sharedInstance.showBottomAlertWithMessage(
```

Comment:
I notice there's not a "Learn more" button like the mocks:

![Screenshot 2025-03-20 at 3 01 07 PM](https://github.com/user-attachments/assets/fa7a5554-0fdb-4632-aba9-07f017a9e3f8)


There's some additional configuration options in `WMFAlertManager` that might help you get that working:
```
...
   dismissPreviousAlerts: true,
   buttonTitle: WMFCommonStrings.learnMoreTitle(),
   buttonCallBack: {code here to navigate to https://www.mediawiki.org/wiki/Help:Temporary_accounts}
)